### DISCUSSION TITLE
Debian installation issue

### DISCUSSION THREAD

**Discussion by srbrodzik** [ARM-DOE/pyart#1179] [Q&A] [answered]
I recently upgraded my Debian system to bullseye (v11) whose default is python 3.9.  However, when trying to install python3-arm-pyart v1.11.2, I get this error:

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages have unmet dependencies:
 python3-arm-pyart : Depends: python3 (< 3.8) but 3.9.2-3 is to be installed
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.

Would installing pyart v1.12.4 solve this problem?

**Comment by zssherman** [comment]
I would try installing the newer pyart version. I believe python 3.9 release was close to or past the time of v1.11.2. After that we did numpy pinning to fix dependency issues. If the newer install doesn't work, let us know! We can try something else.

**Reply by srbrodzik** [reply]
Thanks for the reply.  While I was waiting for a response, I found this in your latest installation directions file on github (https://github.com/ARM-DOE/pyart/blob/main/INSTALL.rst) which implies that installing the newest version of pyart will not solve my problem.  Is that INSTALL file out of date or do I need to downgrade to an older version of python3 to get pyart to work?

Py-ART requires the following software.
Python 3.6.x, 3.7.x or 3.8x
NumPy
SciPy
matplotlib
netCDF4

**Comment by zssherman** [comment, marked as answer]
@srbrodzik That's a good catch we need to update the installation Readme file. We test for python 3.9 and 3.10 and pyart works for both, just need to updated our Readme.

**Reply by srbrodzik** [reply]
Thanks.
